src/modules/random/index.ts

```diff
@@ -23,8 +23,8 @@ export class RandomModule {
   /**
    * Returns a random word.
    *
-   * @see faker.lorem.word()
-   * @see faker.word.sample()
+   * @see faker.lorem.word(): To generate a random placeholder/unreadable word.
```

Comment:
I mean it is very readable if you know how to speak latin.

```suggestion
   * @see faker.lorem.word(): To generate a random placeholder word.
```